deploifai/cli/cloud_profile/create.py

```diff
@@ -124,34 +124,24 @@ def create(context: DeploifaiContextObj, name: str, provider: str):
             click.secho(err, fg="red")
             raise click.Abort()
     elif provider == Provider.AZURE:
-        cloud_credentials["azureSubscriptionId"] = prompt(
-            {
-                "type": "input",
-                "name": "azureSubscriptionId",
-                "message": "Azure Account Subscription ID (We'll keep these secured and encrypted)",
-            }
-        )["azureSubscriptionId"]
-        cloud_credentials["azureTenantId"] = prompt(
-            {
-                "type": "input",
-                "name": "azureTenantId",
-                "message": "Azure Active Directory Tenant ID (We'll keep these secured and encrypted)",
-            }
-        )["azureTenantId"]
-        cloud_credentials["azureClientId"] = prompt(
-            {
-                "type": "input",
-                "name": "azureClientId",
-                "message": "Azure Client ID (We'll keep these secured and encrypted)",
-            }
-        )["azureClientId"]
-        cloud_credentials["azureClientSecret"] = prompt(
-            {
-                "type": "input",
-                "name": "azureClientSecret",
-                "message": "Azure Client Secret / Password (We'll keep these secured and encrypted)",
-            }
-        )["azureClientSecret"]
+        # Get subscription ID
```

Comment:
Should prompt the user which `az account` to use by querying `az account list` and setting it with `az account set --subscription [subscription_id]`